docs/developer.md

```diff
@@ -208,6 +208,13 @@ dlv connect 127.0.0.1:DLV_PORT
 
 ## Unit tests
 
+Prerequisites:
+
+```bash
+make deps
+make mocks
```

Comment:
It seems tp we need to automate a bit `mockgen` installation with for the local env
`go install github.com/golang/mock/mockgen@latest`

Reply:
I'm sorry, I don't quite follow. Are you saying add `go install github.com/golang/mock/mockgen@latest` to these prerequisites?

I notice that `make deps` calls `make tools`, and `make tools` [installs mockgen](https://github.com/zalando/postgres-operator/blob/master/Makefile#L89) version 1.6.0.

Reply:
Oh, yes, sorry missed that deps installing this tools. You are right.